Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+#if os(iOS) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * A user feedback item that serializes to an envelope with the new format described at
+ * https://develop.sentry.dev/application/feedback-architecture/#feedback-events.
+ */
+@objcMembers
+class SentryFeedback: NSObject, SentrySerializable {
+    enum Source: String {
+        case widget
+        case custom
+    }
+
+    var name: String?
+    var email: String?
+    var message: String
+    var hints: [String: Any]?
+    var source: Source
+    let eventId: SentryId
+    
+    /// The event id that this feedback is associated with, like a crash report.
+    var associatedEventId: String?
+
+    init(name: String?, email: String?, message: String, hints: [String: Any]? = nil, source: Source = .widget, associatedEventId: String? = nil) {
+        self.eventId = SentryId()
+        self.name = name
+        self.email = email
+        self.message = message
+        self.hints = hints
+        self.source = source
+        self.associatedEventId = associatedEventId
+        super.init()
+    }
+    
+    func serialize() -> [String: Any] {
```

Comment:
`m`: This still misses a few properties if I'm not mistaken.

Reply:
so the event ID is used to build the envelope header, and i'm not actually sure if/how we are going to use hints for this. I can't remember why hints were even originally included in the API. I'm just going to remove it for now.